docs/source/_toctree.yml

```diff
@@ -14,6 +14,8 @@
       title: Fine-tune BERT for Text Classification on AWS Trainium
     - local: training_tutorials/finetune_llm
       title: Fine-tune Llama 3 8B on AWS Trainium
+    - local: training_tutorials/sft_lora_finetune_llm
+      title: Fine-tune Llama 3 8B on with LoRA and the SFTTrainer
```

Comment:
```suggestion
      title: Fine-tune Llama 3.1 8B with LoRA and the SFTTrainer
```

Reply:
Should we do 3.1?

Reply:
I think we could not do for the rope thing? Or for the transformers version.
Let's keep it like that. In any case we will move to 70B asap, and I can try to do 3.1 then.